src/prompt.rs

```diff
@@ -40,3 +51,49 @@ fn execute(args : Vec<&str>) -> io::Result<i32> {
             Ok(result.code().unwrap_or(0))
         })
 }
+
+#[cfg(test)]
+mod tests {
+    use std::env;
+    use std::path::PathBuf;
+    use super::{get_prompt_string, WAVE_EMOJI};
+
+    fn prompt_unset_pwd() {
+        println!("RUN_TEST_TASKS: {:?}", env::var("RUN_TEST_TASKS"));
```

Comment:
That actually shouldn't be there.  It prints the debug view, in this case because the return value is an option.  I was just doing some debugging and forgot to take this out.  I'll remove it